packages/usa-form/src/styles/_usa-form.scss

```diff
@@ -17,41 +17,12 @@
     text-decoration: none;
   }
 
-  .usa-input,
   .usa-range,
-  .usa-select,
-  .usa-textarea {
+  .usa-select {
```

Comment:
> [!note]
> Removed these elements because the specificity of this rule caused an [override](https://github.com/uswds/uswds/pull/5935#discussion_r1853057805) of the `usa-input--[width]` classes when they were used inside `.usa-form`. We should test that there are no regressions as the result of this change.

Reply:
**question:** Should we also remove `usa-range` and `usa-select` for uniformity?

Reply:
> **thought:** This might make it harder to customize entire forms.
> 
> Currently, setting `max-width: none` here allows the inputs to match the max width of the form element. Removing this means the form and inputs have separate max-widths. I'm not sure if this should be a blocker or not but I'm curious how this might affect downstream users.

This is a fair note. This update would cause a regression in:
- `.usa-form` child inputs if `$theme-input-max-width` is set to a width narrower than "mobile"
- `.usa-form--large` child inputs if `$theme-input-max-width` is set to a width narrower than "mobile-lg"

**Preview of regression:**
![image](https://github.com/user-attachments/assets/6c0872a8-29f2-41d9-b57c-8600e45a72a2)

This is because the max-width for `usa-form` in desktop widths is set to ["mobile"/20rem](https://github.com/uswds/uswds/blob/develop/packages/usa-form/src/styles/_usa-form.scss#L12) and `usa-form--large` is ["mobile-lg"/ 30rem](https://github.com/uswds/uswds/blob/develop/packages/usa-form/src/styles/_usa-form.scss#L71). By default, `$theme-input-max-width` is set to `mobile-lg`, so it is either the same width or wider than the `.usa-form` elements. But if the input width is set to less than that, we start to see regressions. 

For this reason, I'm thinking we restore this `max-width: none` rule in `usa-form` and instead explore alternatives, either reducing specificity in this rule or increasing specificity in the `usa-input` and `usa-input-prefix-suffix` packages. I'll come back with solutions.

Reply:
Update: proposing using `:where()` here instead of removing the rules entirely. This will preserve the the `usa-form` max-width on child input elements in the default state, but will also allow a custom max width `usa-input--[width]` and `usa-input-group--[width]` classes. 

Curious what you think.